tikv-configuration-file.md

```diff
@@ -737,6 +737,19 @@ Configuration items related to Raftstore.
 + Default value: `"30s"`
 + Minimum value: `0`
 
+### `max-apply-unpersisted-log-limit` <span class="version-mark">New in v8.1.0</span>
+
+> **Warning:**
+>
+> This feature is experimental. It is not recommended that you use it in the production environment.
+
++ The maximum number of committed but not persisted Raft logs that can be applied.
+
+    + Setting this configuration item to a value greater than `0` enables the TiKV node to apply committed but not persisted Raft logs in advance, effectively reducing long-tail latency caused by IO jitter on that node. However, it might also increase the memory usage of TiKV and the disk space occupied by Raft logs.
```

Comment:
```suggestion
    + Setting this configuration item to a value greater than `0` enables the TiKV node to apply committed but not persisted Raft logs in advance, effectively reducing long-tail latency caused by IO jitter on that node. However, it might also increase the memory usage of TiKV and the disk space occupied by Raft logs.
    + Setting this configuration item to `0` disables this feature, meaning that TiKV must wait until Raft logs are both committed and persisted before applying them. This behavior is consistent with the behavior before v8.2.0.
```